tests/pages/organization-index.js

```diff
@@ -0,0 +1,8 @@
+import {
+  create,
+  visitable
+} from 'ember-cli-page-object';
+
+export default create({
+  visit: visitable(':organization')
+});
```

Comment:
These and the other page objects are a good start, but there's much more cool stuff you can do with page objects beyond setting up a visit call. All the UIcan be encapsulated in here as well. Take a look at https://github.com/code-corps/code-corps-ember/blob/develop/tests/pages/onboarding.js for some samples.